soh/soh/Enhancements/randomizer/3drando/hint_list.cpp

```diff
@@ -2752,6 +2752,9 @@ void StaticData::HintTable_Init() {
                                                 /*french*/ "As-tu récemment ressenti une vague de #puissance magique#? Un mystérieux hibou m'a dit  qu'elle provenait du #[[1]]#. Tu devrais aller y jeter un coup d'oeil, @!\x0B",
                                                            {QM_GREEN, QM_RED}, {}, TEXTBOX_TYPE_BLUE));
 
+    hintTextTable[RHT_LOACH_HINT] = HintText(CustomMessage("What?^You wanna know about the&%rHyrule Loach%w?^It's a big fish, but it's so rare that&I'll give my %g[[1]]%w&to anyone who catches it. Seriously!",
```

Comment:
Every other hint still uses `[[1]]` tho.

Reply:
wait i had that backwards, damn rustiness, something else must have caused it to fail then

Reply:
Real problem, hints have auto-format now so do not need as much new line formatting, It can actually cause problems in some instances. I recommend running a few seeds and seeing how the text generates to avoid awkward line breaks like this.
![image](https://github.com/HarbourMasters/Shipwright/assets/93387759/05856492-2e7a-4fa0-9d61-abdb18e57368)

Reply:
I think I fixed it, I can't test it right now due to a mysterious crash